firecracker/rootfs/docker_runtime/create_rootfs_image.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+set -euo pipefail
+set -o xtrace
+
+readonly IMAGE="/tmp/rootfs.ext4"
+readonly MOUNT_POINT="/tmp/rootfs-mount"
+readonly DIST_FILE="/dist/firecracker_rootfs.ext4.tar.gz"
+
+########################################
+# Create image and mount it.
+########################################
+# make a 50mb empty file
+dd if=/dev/zero of="${IMAGE}" bs=1M count=50
+
+# format that filesystem
+/sbin/mkfs.ext4 "${IMAGE}"
+
+# make a mount-point
+mkdir "${MOUNT_POINT}"
+
+# Mount
+mount -t fuse-ext2 -o rw+ "${IMAGE}" "${MOUNT_POINT}"
```

Comment:
ext2?

Reply:
yeah believe it or not 
![image](https://user-images.githubusercontent.com/69007229/156246531-ac0e33e2-a3b4-4657-aa1e-306a40814a61.png)